cmd/apps.go

```diff
@@ -17,8 +17,8 @@ import (
 )
 
 // AppCreate creates an app.
-func AppCreate(id string, buildpack string, remote string, noRemote bool) error {
-	s, err := settings.Load()
+func AppCreate(cf, id, buildpack, remote string, noRemote bool) error {
+	s, err := settings.Load(cf)
```

Comment:
We should have a settings struct that accepts a client file location and has a Load() function. That way we don't have to pass around a string to the client file on every client command.